Added valid prefixes to Release Notes (#1041)
demisto/content-docs: docs/documentation/release-notes.md

```markdown
#### Integrations
##### Cortex XDR - IR

> Release note here.

Reviewer: @tschanfeld please revert this change to be a list (`- Release note here.`), as it matters for automated tests this document describes.

Reviewer: @kgal-pan please check whether this passes validations. If it does not, let's change it back.

```

For single line RNs with a nested list, follow this format:
```markdown
#### Integrations
##### Cortex XDR - IR
Release note here.
  - List item 1

>   - List item 2

Reviewer: @tschanfeld please revert this change to be a nested list (`- Release note here.`)

Reviewer: @kgal-pan please check which of the following works with `validate`:
```
- a
  - b
  - c
```
or
```
a 
  -  b
  - c
```

Author: @dorschw - Looks like both syntaxes pass validation:

```bash

> file=Packs/SecurityScorecard/ReleaseNotes/1_0_4.md
```

```bash
# Hyphen in root
> cat $file&&demisto-sdk validate -i $file --silence-init-prints

#### Integrations
##### SecurityScorecard
- Release note here
        - Fixed an issue with the *get_last_run* unit test.
        - Updated the Docker image to: *demisto/python3:3.10.4.28442*.
You are using demisto-sdk 1.6.4.

================= Validating file /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md =================

Validating /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md as releasenotes

The files are valid
```
```bash
# No hyphen in root
> cat $file&&demisto-sdk validate -i $file --silence-init-prints

#### Integrations
##### SecurityScorecard
Release note here
        - Fixed an issue with the *get_last_run* unit test.
        - Updated the Docker image to: *demisto/python3:3.10.4.28442*.
You are using demisto-sdk 1.6.4.

================= Validating file /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md =================

Validating /Users/kgal/dev/demisto/content/Packs/SecurityScorecard/ReleaseNotes/1_0_4.md as releasenotes

The files are valid
```

    - Bad example: `Added the timeout parameter.`  
    - Good example: `Added the timeout parameter, which enables you to define the amount of time (in minutes) that the integration will try to execute commands before it throws an error.`

> - If this is a single line release note, there is no need for the bullet point, just a regular sentence.

Reviewer: ```suggestion
- Single line release notes do not need a bullet point.
```

> - The release note line must start with one of the following prefixes:

Reviewer: ```suggestion
- Release notes must start with one of the following prefixes:
```

```

> 'Added support for...'
> 'Added the...'
> 'Fixed an issue...'
> 'Improved implementation...'
> 'Updated the Docker image to...'
> 'You can now...'
> 'Deprecated. '
> 'Deprecated the...'

Reviewer: ```suggestion
'Added support for '
'Added the '
'Fixed an issue '
'Improved implementation '
'Updated the Docker image to '
'You can now '
'Deprecated. '
'Deprecated the' 
```

as the `...` are not part of the prefix

```

> Specifying any other prefix will result in an error when running `demisto-sdk doc-review`:

Reviewer: ```suggestion
Release notes not using the prefixes will result in an error when running `demisto-sdk doc-review`:
```
